packages/lib-user/src/components/shared/Layout/Layout.stories.js

```diff
@@ -0,0 +1,67 @@
+import { Layer, Link, SettingsOption } from 'grommet-icons'
+
+import Layout from './Layout'
+import HeaderLink from '../HeaderLink'
+import ContentBox from '../ContentBox'
+import MainContent from '../MainContent'
+import HeaderToast from '../HeaderToast'
+import HeaderButton from '../HeaderButton'
+
+export default {
+  title: 'Components / Shared / Layout',
+  component: Layout
+}
+
+const pageContent = (
+  <ContentBox>
+    <MainContent />
+  </ContentBox>
```

Comment:
```suggestion
  <MainContent />
```

The MainContent component isn't usually wrapped with a ContentBox component.